
* So we've accounted for all of the appropriate uses of our functions, but what happens if we start to think about how it could break?  As we noted earlier, doubler('2') may or may not be very helpful. 
* Let's try a test-driven development approach.
* Assuming we wanted to add input validation to the doubler function and throw an exception if we don't receive an integer

